tests/rules/method.js

```diff
@@ -1056,5 +1060,15 @@ eslintTester.run("method", rule, {
                 },
             ],
         },
+        {
+            code: "document.write(badness);",
```

Comment:
This does not fulfill the requirements. I would like a violation that is new and uses the "as" expression

Reply:
`document.write(evil as String)` maybe?